pkg/component/iptables/iptables.go

```diff
@@ -32,6 +38,81 @@ const (
 	ModeLegacy = "legacy"
 )
 
+type IPTables struct {
```

Comment:
We could also name this `Component` to make it clear that this is the component: `iptables.Component` reads a bit nicer than `iptables.IPTables` IMO.